digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/client/vvh/RoadLinkClient.scala

```diff
@@ -37,10 +39,15 @@ object FeatureClass {
   case object AllOthers extends FeatureClass
 }
 
+sealed trait IRoadLinkFetched extends RoadLinkLike {
+  def featureClass: FeatureClass
+  def modifiedAt: Option[DateTime]
+}
```

Comment:
Mistä IRoadLinkFetched nimi tulee ja mihin ensimmäinen i viittaa?

Reply:
RoadlinkFethed sana oli varattu ja I viittaa että kyseessä on interface, en nyt keksinyt parempaa nimeä kun tän pitäisi toimia RoadLinkFetched ja HistoriaRoadlink kanssa, Niillä ei nyt ole mitään oikeaa kosketustpintaa mut käytetään vähän samalla tasolla koodissa joten vedin näin